en/docs/guides/mfa/totp-config-advanced.md

```diff
@@ -0,0 +1,115 @@
+# TOTP Configurations
+
+This page lists out all the advanced configurations related to TOTP. 
+
+The following code snippet shows a sample TOTP configuration in the `<IS_HOME>/repository/conf/deployment.toml` file. 
+
+```toml
+[authentication.authenticator.totp.parameters]
+encodingMethod="Base32"
+timeStepSize="30"
+windowSize="3"
+authenticationMandatory=true
+enrolUserInAuthenticationFlow=true
+usecase="local"
+secondaryUserstore="primary"
+TOTPAuthenticationEndpointURL="totpauthenticationendpoint/totp.jsp"
+TOTPAuthenticationEndpointErrorPage="totpauthenticationendpoint/totpError.jsp"
+TOTPAuthenticationEndpointEnableTOTPPage="totpauthenticationendpoint/enableTOTP.jsp"
+Issuer="WSO2"
+UseCommonIssuer=true
+```
+
+The parameter values given above show the **default** configurations in WSO2 Identity Server. 
+
+If you wish to change a parameter value to something other than the default value, add the configuration to the `deployment.toml` file using the following format.
+
+```toml
+[authentication.authenticator.totp.parameters] 
+<Property-name> = <Property-value> 
+```
+
+----
+
+#### encodingMethod
+
+The encoding method which is used to generate the TOTP.
+
+----
+
+#### timeStepSize 
+
+The time step size, which is used to validate the TOTP.
```

Comment:
I don't think we need the comma here since this is not additional information. WDYT? 
If yes, please change in all places.